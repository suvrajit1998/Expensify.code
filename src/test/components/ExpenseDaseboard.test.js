import React from "react";
import { shallow } from "enzyme";
import Examlemainpage from "../../components/examlemainpage";

test("Should render DaseBoardPage", () => {
  const wrapper = shallow(<Examlemainpage />);

  expect(wrapper).toMatchSnapshot();
});
