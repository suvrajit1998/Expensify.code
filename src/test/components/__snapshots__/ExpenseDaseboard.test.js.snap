// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render DaseBoardPage 1`] = `
<div>
  This is the Main Page Content
  <Connect(expenselistfilters) />
  <Connect(ExpensesList) />
</div>
`;
